README.md

```diff
@@ -11,46 +11,46 @@ This is a collection of projects open to new lead maintainers. The projects may
 
 ## Project List
 
-#### [Betty](https://github.com/pickhardt/betty)
+#### [Betty](https://github.com/pickhardt/betty) [ruby]
 Friendly English-like interface for your command line. Written in Ruby.
 
-#### [forem](https://github.com/rubysherpas/forem)
+#### [forem](https://github.com/rubysherpas/forem) [ruby]
 Rails 3 and Rails 4 forum engine.
 
-#### [gitx](https://github.com/pieter/gitx)
+#### [gitx](https://github.com/pieter/gitx) [objective-c]
 GitX is a git GUI made for Mac OS X.
 
-#### [Her](https://github.com/remiprev/her/)
+#### [Her](https://github.com/remiprev/her/) [ruby]
 ORM (Object Relational Mapper) that maps REST resources to Ruby objects.
 
-#### [Hermes](https://github.com/HermesApp/Hermes)
+#### [Hermes](https://github.com/HermesApp/Hermes) [objective-c]
 Compact OS X Pandora client that doesn't use Flash.
 
-#### [jQuery Modal](https://github.com/kylefox/jquery-modal)
+#### [jQuery Modal](https://github.com/kylefox/jquery-modal) [js]
 The simplest possible modal for jQuery.
 
-#### [jQuery Tablesort](https://github.com/kylefox/jquery-tablesort)
+#### [jQuery Tablesort](https://github.com/kylefox/jquery-tablesort) [js]
 A tiny & dead-simple jQuery plugin for sortable tables
 
-#### [Papa Parse](https://github.com/mholt/PapaParse)
+#### [Papa Parse](https://github.com/mholt/PapaParse) [js]
 High-performance CSV parser for the browser. Written in JavaScript.
 
-#### [polipo](https://github.com/jech/polipo)
+#### [polipo](https://github.com/jech/polipo) [c language]
 Caching HTTP proxy. Written in C.
 
-#### [python-mode](https://github.com/klen/python-mode)
+#### [python-mode](https://github.com/klen/python-mode) [Python]
```

Comment:
:+1: for this change. However, I'm curious why Python is the only language that starts with an uppercase letter.